### ISSUE TITLE
[CoE Starter Kit - QUESTION] CoE Setup and Upgrade Wizard Office365Groups error

### ISSUE THREAD

**Issue by KoshiroTosa** [microsoft/coe-starter-kit#7041]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

The CoE Setup and Upgrade Wizard detected an Office365Groups error.
It has been showing up since I opened this application.
I am not sure if this error is the cause, but I am unable to set up an admin persona. Also, the target group is not output in the setup group's administrator persona choices.
Do you know the cause of this?

The Global Administrator role in Microsoft Entra is assigned to the user account.

![画像1](https://github.com/microsoft/coe-starter-kit/assets/150125365/bade7988-346e-4189-994a-23ec31a24bac)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

_4.16_

### What app or flow are you having the issue with?

_CoE Setup and Upgrade Wizard_

### What method are you using to get inventory and telemetry?

None

AB#749

**Comment by Jenefer-Monroe** [comment]
If you are adding the admin without the drop down, be sure to enter them with their email (not their GUID or display name)
![Image](https://github.com/microsoft/coe-starter-kit/assets/89584709/b226a2ee-3c1d-41fe-a9b3-46082f3e2d6b)

Note that the other env vars (maker, user) its the GUID you enter
![Image](https://github.com/microsoft/coe-starter-kit/assets/89584709/74abf426-e9ab-48a7-89ca-cf16d282d411)

**Comment by KoshiroTosa** [comment]
Thank you for your response.
I have tried entering the email address for the group, but the group is not selected.
![画像2](https://github.com/microsoft/coe-starter-kit/assets/150125365/ad55a080-fc4c-48e1-bbaa-392169ac80b8)

I keep getting an Office365Gruops error when I start the CoE Setup and Upgrade Wizard.
![画像1](https://github.com/microsoft/coe-starter-kit/assets/150125365/1a464f2e-957c-4ebc-bbf1-e0b37b40d6a4)

What is the cause of this Office365Gruops error?

**Comment by tomka75** [comment]
Same problem here with the setup wizzard. Does not show the pre created M365 group, does not accept the email nor the ID. "Select group" and "Cancel" button do not work. We configured makers and users manually, but what env. variable is used to store the admin group id?

**Comment by Jenefer-Monroe** [comment]
> Same problem here with the setup wizzard. Does not show the pre created M365 group, does not accept the email nor the ID. "Select group" and "Cancel" button do not work. We configured makers and users manually, but what env. variable is used to store the admin group id?

When you say its the same problem, do you mean its the same error message? If not, please post a new issue for us with some screenshots showing what you are seeing.

**Comment by tomka75** [comment]
![image](https://github.com/microsoft/coe-starter-kit/assets/32197566/4e4b477c-7711-4c1f-bda9-f824045e27f4)
Find items finds groups, but not that one we need (its a M365 group). Adding the group eMail address and clicking on Cancel or Select group does not work, buttons are not reacting. We do not get the ListGroups error.

**Comment by Jenefer-Monroe** [comment]
ok please do post that separately. Its a different issue that op is experiencing. I'll go ahead and hide the above so that we can get his unblocked
Thank you!

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
